.rakeTasks

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
What is this, and what is it for?

Reply:
This comes with the project when I pulled it and started to work, really I don't know why it is being used

Reply:
Included into .gitignore, hence this will not appear again.

Reply:
I think you still need to `git rm` this